cvat/apps/documentation/installation.md

```diff
@@ -306,3 +307,299 @@ volumes:
+b85497c44836        cvat_cvat_ui          "nginx -g 'daemon of…"   About a minute ago   Up About a minute   80/tcp                                     cvat_ui
+d25a00475849        cvat                  "/usr/bin/supervisord"   About a minute ago   Up About a minute   8080/tcp, 8443/tcp                         cvat
+6353a43f55c3        redis:4.0-alpine      "docker-entrypoint.s…"   About a minute ago   Up About a minute   6379/tcp                                   cvat_redis
+52009636caa8        postgres:10-alpine    "docker-entrypoint.s…"   About a minute ago   Up About a minute   5432/tcp                                   cvat_db
+```
+
+We will attach `cvat_proxy` container to run `acme.sh` scripts.
+
+```bash
+admin@tempVM:~/cvat$ docker exec -ti cvat_proxy /bin/sh
+
+# install some missing software inside cvat_proxy
+/ # apk add openssl curl
+/ # curl https://get.acme.sh | sh
+/ # ~/.acme.sh/acme.sh -h
+[... many lines ...]
+
+/ # ~/.acme.sh/acme.sh --issue -d my-cvat-server.org -w /letsencrypt
+[Fri Apr  3 20:49:05 UTC 2020] Create account key ok.
+[Fri Apr  3 20:49:05 UTC 2020] Registering account
+[Fri Apr  3 20:49:06 UTC 2020] Registered
+[Fri Apr  3 20:49:06 UTC 2020] ACCOUNT_THUMBPRINT='tril8-LdJgM8xg6mnN1pMa7vIMdFizVCE0NImNmyZY4'
+[Fri Apr  3 20:49:06 UTC 2020] Creating domain key
+[ ... many more lines ...]
+[Fri Apr  3 20:49:10 UTC 2020] Your cert is in  /root/.acme.sh/my-cvat-server.org/my-cvat-server.org.cer 
+[Fri Apr  3 20:49:10 UTC 2020] Your cert key is in  /root/.acme.sh/my-cvat-server.org/my-cvat-server.org.key 
+[Fri Apr  3 20:49:10 UTC 2020] The intermediate CA cert is in  /root/.acme.sh/my-cvat-server.org/ca.cer 
+[Fri Apr  3 20:49:10 UTC 2020] And the full chain certs is there:  /root/.acme.sh/my-cvat-server.org/fullchain.cer 
+
+/ # cp ~/.acme.sh/my-cvat-server.org/my-cvat-server.org.cer /letsencrypt/certificate.cer
+/ # cp ~/.acme.sh/my-cvat-server.org/my-cvat-server.org.key /letsencrypt/certificate.key
+/ # cp ~/.acme.sh/my-cvat-server.org/ca.cer /letsencrypt/ca.cer
+/ # cp ~/.acme.sh/my-cvat-server.org/fullchain.cer /letsencrypt/fullchain.cer
+/ # exit
+admin@tempVM:~/cvat$ ls letsencrypt/
+ca.cer  certificate.cer  certificate.key  fullchain.cer
+admin@tempVM:~/cvat$ mkdir cert
+admin@tempVM:~/cvat$ mv letsencrypt/* ./cert
+```
+
+##### 4. Reconfigure Nginx for HTTPS access
+
+Update Docker Compose configuration to mount the certificate directory.
+
+```yml
+# docker-compose.override.yml
+version: "2.3"
+
+services:
+  cvat_proxy:
+    environment:
+      CVAT_HOST: my-cvat-server.org
+    ports:
+    - "443:443"
+    volumes:
+    - ./letsencrypt:/letsencrypt
+    - ./cert:/cert:ro # this is new
+    
+  cvat:
+    environment:
+      ALLOWED_HOSTS: '*'
+```
+
+Also, reconfigure Nginx to use `443/tcp` and point it to the new keys.
+
+```bash
+server {
+    listen       80;
+    server_name  _ default;
+    return       404;
+}
+
+server {
+    listen       443 ssl;
+    server_name  ${CVAT_HOST};
```

Comment:
Could you please also add 301 redirect to https:// for this server_name on 80 port?